sql/hive/src/main/scala/org/apache/spark/sql/hive/orc/OrcFileFormat.scala

```diff
@@ -256,9 +257,6 @@ private[orc] class OrcOutputWriter(
 }
 
 private[orc] object OrcRelation extends HiveInspectors {
```

Comment:
@dongjoon-hyun, mind changing this to `OrcFileFormat` while we are here (see https://github.com/apache/spark/pull/14529)?

Reply:
Sure. No problem. Thank you for review, @HyukjinKwon !